vuepress/docs/next/tutorials/consume/cds.md

```diff
@@ -20,25 +20,24 @@ A set of resources are necessary to separate the storage and user data for the p
 4. Replace the placeholders in `entando-cds.yaml` with the appropriate values for your environment. 
 
 Conventions:
-* The storage limit and request is set to 1Gi and can be modified on the persistent volume claim
-* The file upload size limit is set to 150m and can be configured via the ingress annotations
-* In order to enable TLS, add a TLS secret and configure it on the ingress. Note that public URLs (e.g., `CDS_PUBLIC_URL`) should use the same protocol (`http` or `https`) as the Entando Application. Private/cluster-level URLs (e.g., `CDS_PRIVATE_URL`) should use `http`. 
+* The storage limit and request is set to 1Gi and can be modified on the persistent volume claim.
+* The file upload size limit is set to 150m and can be configured via the ingress annotations.
+* In order to enable TLS, add a TLS secret and configure it on the ingress. Note that public URLs (e.g., `CDS_PUBLIC_URL`) should use the same **http** or **https** protocol as the Entando Application. Private/cluster-level URLs (e.g., `CDS_PRIVATE_URL`) should use **http**. 
+* The same Keycloak public key can be used if all the tenants share a Keycloak instance using different realms.
 
 | Placeholder | Description 
 |:--|:--
 | YOUR-APP-NAME | The name of the application, e.g., quickstart
 | YOUR-HOST-NAME | The base host name of the application, e.g., your-domain.com
-| YOUR-TENANT-ID | The identifier for the tenant, e.g., mysite1 
+| YOUR-TENANT-ID | The identifying name of the current tenant and a subdomain name, e.g., yoursite results in yoursite.your-domain.com for the base url of the tenant
```

Comment:
This isn't exactly right since the user can choose whether or not to make it a subdomain when they apply the yaml files. My only question is whether this is the right place for this note since it's a more general statement. Maybe we need to do it consistently in each of the docs, for sure on the parent one.
```suggestion
| YOUR-TENANT-ID | The identifying name of the current tenant. In most cases it will also be used to determine the base URL of the tenant. For example, yoursite results in yoursite.your-domain.com.
```